csharp/ql/test/library-tests/csharp7/Discards.expected

```diff
@@ -6,3 +6,4 @@
 | CSharp7.cs:224:17:224:17 | _ | Double |
 | CSharp7.cs:224:28:224:28 | _ | Boolean |
 | CSharp7.cs:225:10:225:10 | _ | Int32 |
+| CSharp7.cs:267:25:267:25 | _ | null |
```

Comment:
Why `null` and not `Double`?

Reply:
Roslyn said it was `null`. I don't think this is worth fixing.